README.md

```diff
@@ -46,7 +47,7 @@ This will launch a new Firefox browser with the extension installed. You should
 
 You may face errors on performing ```npm install``` that can be resolved by updating the node to its latest version [see here](https://www.hostingadvice.com/how-to/update-node-js-latest-version/)
 
-If a new browser does not open, it might be because the path to Nightly is not found. Use the command `FIREFOX="/usr/bin/firefox" npm start` instead.
+If a new browser does not open, it might be because the path to Nightly is not found(most probable case). Find your binary file in the system, and enter the entire path to it. Binary file is the application through which firefox nightly can be opened manually. Use the command `FIREFOX="/usr/bin/firefox" npm start` instead.
```

Comment:
Oops, you are missing a space before the parenthesis. It might be clearer if you say "Find your application's executable: the file where Firefox Nightly can be opened manually. Enter the entire path of the file"